docs/fastdds/api_reference/dds_pim/core/Time_t/Time_t.rst

```diff
@@ -0,0 +1,10 @@
+.. _api_pim_time_t:
+
+.. rst-class:: api-ref
+
+DDS Layer Time_t
+----------------
+
+.. doxygenstruct:: eprosima::fastdds::dds::Time_t
+    :project: FastDDS
+    :members:
```

Comment:
[Not found](https://eprosima-fast-rtps--870.org.readthedocs.build/en/870/fastdds/api_reference/dds_pim/core/Time_t/Time_t.html)